gallery/tutorials/pipeline_demo.py

```diff
@@ -216,13 +233,13 @@
 
 from aspire.abinitio import CLSyncVoting
 from aspire.source import OrientedSource
+from aspire.utils import Rotation
 
 # Stash true rotations for later comparison
-true_rotations = src.rotations[:n_classes]
+true_rotations = Rotation(src.rotations[:n_classes])
 
-# For this low resolution example we will customize the ``CLSyncVoting``
-# instance to use fewer theta points ``n_theta`` then the default value of 360.
-orient_est = CLSyncVoting(avgs, n_theta=72)
+# Instantiate a ``CLSyncVoting`` object for estimating orientations.
+orient_est = CLSyncVoting(avgs)
 
 # Instantiate an ``OrientedSource``.
 oriented_src = OrientedSource(avgs, orient_est)
```

Comment:
I think it might be better to move a rotation estimation here so that the CL output is in this block instead of the mean angular distance block.